packages/cli/bin/expf.mjs

```diff
@@ -0,0 +1,68 @@
+import { parseArgs } from 'node:util';
+import { argv } from 'node:process';
+
+const { values, positionals } = parseArgs({
+  args: argv,
+  allowPositionals: true,
+  options: {
+    help: {
+      type: 'boolean'
+    },
+    cwd: {
+      type: 'string'
+    },
+
+    repo: {
+      type: 'string'
+    },
+    'repo-ref': {
+      type: 'string'
+    },
+
+    runner: {
+      type: 'string',
+      short: 'u'
+    },
```

Comment:
These flags don't apply to all commands. We might want to think about doing some added validation on this, but I don't want to work on those details before landing this PR. So if no one objects, we can just leave this as is for now and track it in an issue.